hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -294,7 +291,7 @@ public ReentrantReadWriteLock getTableLock(String tableName) {
   private final long omEpoch;
 
   private Map<String, Table> tableMap = new HashMap<>();
-  private List<TableCacheMetrics> tableCacheMetrics = new LinkedList<>();
+  private Map<String, TableCacheMetrics> tableCacheMetricsMap = new HashMap<>();
```

Comment:
```suggestion
  private final Map<String, TableCacheMetrics> tableCacheMetricsMap = new HashMap<>();
```

Reply:
Thanks @adoroszlai for review. I have done the change.